src/common/menu.rs

```diff
@@ -57,19 +58,15 @@ pub unsafe fn write_menu() {
         .join(format!("{program_id:016X}"))
         .join(format!("manual_html/html-document/{htdocs_dir}.htdocs/"))
         .join("training_menu.html");
-
-    let write_resp = fs::write(menu_html_path, data);
-    if write_resp.is_err() {
-        println!("Error!: {}", write_resp.err().unwrap());
-    }
+    fs::write(menu_html_path, data).expect("Failed to write menu HTML file");
```

Comment:
If we panic, doesn't the plugin crash here? I think we still want to print the error and continue.

Reply:
Oops, sorry. Was incorrectly assuming what this code was doing!